src/test/java/command/AddEventCommandTest.java

```diff
@@ -0,0 +1,34 @@
+package command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class AddEventCommandTest {
+
+
+
+    @Test
+    public void executeCommand_missingDescription_exceptionThrown() {
+        DukeException exception = assertThrows(DukeException.class, () -> {new AddEventCommand("event ").excuteCommand(new TaskList(), new Ui(), new Storage("duke"));});
+        assertEquals("Missing the description!", exception.getMessage());
+    }
+
+
+    @Test
+    public void executeCommand_missingTwoDateTime_exceptionThrown() {
+        DukeException exception = assertThrows(DukeException.class, () -> {new AddEventCommand("event meeting").excuteCommand(new TaskList(), new Ui(), new Storage("duke"));});
+        assertEquals("Invalid format for new Event!", exception.getMessage());
+    }
+
+    @Test
+    public void executeCommand_missingOneDateTime_exceptionThrown() {
+        DukeException exception = assertThrows(DukeException.class, () -> {new AddEventCommand("event meeting /from 2024-10-01 23:59").excuteCommand(new TaskList(), new Ui(), new Storage("duke"));});
```

Comment:
I like how thorough you are with testing! However the lines where "exception" variable are is longer than 120 characters in length which is the java coding standards, I'd recommend putting in some line breaks, maybe before "() ->". Other than that it looks good!